eve/workers/build/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM ubuntu:xenial
+
+#
+# Install packages needed by the buildchain
+#
+ENV LANG C.UTF-8
+COPY ./s3_packages.list ./buildbot_worker_packages.list /tmp/
+RUN apt-get update \
+    && cat /tmp/*packages.list | xargs apt-get install -y \
+    && wget https://nodejs.org/dist/v6.9.5/node-v6.9.5-linux-x64.tar.xz \
```

Comment:
Can we try using `n` to manage node.js distribution? This way we can automatically install `n 6.x` and get the latest minor-minor version.
https://github.com/tj/n

Reply:
Yeah that's a good idea. I'll add that in.